src/Raven.Client/Documents/Subscriptions/SubscriptionWorkerOptions.cs

```diff
@@ -110,6 +110,7 @@ public SubscriptionWorkerOptions(string subscriptionName) : this()
                 throw new ArgumentException("Value cannot be null or empty.", nameof(subscriptionName));
 
             SubscriptionName = subscriptionName;
+            WorkerId = Guid.NewGuid().ToString();
```

Comment:
it should be move to private ctor?

Reply:
we don't want to generate `WorkerId` upon de-serializing.